src/main/scala/org/broadinstitute/dsde/firecloud/webservice/NotificationsApiService.scala

```diff
@@ -3,14 +3,27 @@ package org.broadinstitute.dsde.firecloud.webservice
 import org.broadinstitute.dsde.firecloud.FireCloudConfig
 import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectives, PerRequestCreator}
 import org.broadinstitute.dsde.firecloud.utils.StandardUserInfoDirectives
+import spray.http.{HttpMethods, Uri}
 import spray.routing._
 
 trait NotificationsApiService extends HttpService with PerRequestCreator with FireCloudDirectives with StandardUserInfoDirectives {
+  private final val ApiPrefix = "api/notifications"
+  private final val General = "general"
+  private final val Workspace = "workspace"
 
-  private final val ApiPrefix = "notifications"
-
-  val notificationsRoutes: Route =
-    pathPrefix("api") {
-      passthroughAllPaths(ApiPrefix, FireCloudConfig.Rawls.notificationsUrl)
+  final val notificationsRoutes: Route = {
+    get {
+      pathPrefix(separateOnSlashes(ApiPrefix)) {
+        unmatchedPath { remaining =>
+          val encodedTargetUri = Uri(encodeUri(FireCloudConfig.Rawls.notificationsUrl + remaining))
+          path(General) {
+            passthrough(encodedTargetUri, HttpMethods.GET)
+          } ~
+          path(Workspace / Segment / Segment) { (_, _) =>
+            passthrough(encodedTargetUri, HttpMethods.GET)
+          }
```

Comment:
Instead of
```
path(General) {
  passthrough(encodedTargetUri, HttpMethods.GET)
} ~
path(Workspace / Segment / Segment) { (_, _) =>
  passthrough(encodedTargetUri, HttpMethods.GET)
}
```
I wanted to have something like 
```
path(General) | path(Workspace / Segment / Segment) {
  passthrough(encodedTargetUri, HttpMethods.GET)
}
```
but types don't work out. I'm guessing it's possible to have a `regex` on the URIs but not sure if it's worth it.

@davidangb